cli/main.rs

```diff
@@ -297,6 +312,14 @@ fn print_cache_info(
       colors::bold("Language server registries cache:"),
       registry_cache,
     );
+    println!("{} {:?}", colors::bold("Origin storage:"), origin_dir);
+    if location.is_some() {
+      println!(
+        "{} {:?}",
+        colors::bold("Local Storage:"),
```

Comment:
nit: is this capitalization intentional?

Reply:
Yes, as "Local Storage" is a proper noun